app/controllers/event1/UnauthPaymentAndSanctionChargesController.scala

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.event1
+
+import controllers.actions._
+import models.enumeration.EventType
+import pages.Waypoints
+import play.api.i18n.{I18nSupport, MessagesApi}
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
+import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendBaseController
+import views.html.event1.UnauthPaymentAndSanctionChargesView
+
+import javax.inject.Inject
+
+class UnauthPaymentAndSanctionChargesController @Inject()(
+                                           override val messagesApi: MessagesApi,
+                                           identify: IdentifierAction,
+                                           getData: DataRetrievalAction,
+                                           requireData: DataRequiredAction,
+                                           val controllerComponents: MessagesControllerComponents,
+                                           view: UnauthPaymentAndSanctionChargesView
+                                         ) extends FrontendBaseController with I18nSupport {
+
+  private val eventType = EventType.Event1
```

Comment:
If always Event1, could we rename val or just use EventType.Event1 where needed?

Reply:
I don't think it matters too much in terms of efficiency etc. But if we want to do this then it would be better to do it on a separate ticket as it will apply to other pages as well (and also we could change the g8 templates).